README.adoc

```diff
@@ -139,6 +139,7 @@ Translations of the guide are available in the following languages:
 * https://github.com/theSkilled/clojure-style-guide/blob/pt-BR/README.md[Portuguese] (Under progress)
 * https://github.com/Nondv/clojure-style-guide/blob/master/ru/README.md[Russian]
 * https://github.com/jeko2000/clojure-style-guide/blob/master/README.md[Spanish]
+* https://github.com/LeaveNhA/clojure-style-guide/blob/master/README.md[Turkish]
```

Comment:
Probably you meant to put `README.adoc` here? The other links are outdated, as they are from the time before the conversion to AsciiDoc.

Reply:
I thought that there is an unwritten rule or some conversion to create translation/content.
I can change my repository to serve an ASCIIDoc file. Thank you for pointing out.